docs/docs/features/startup-bootloader-key.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Startup Bootloader Key
+sidebar_label: Startup Bootloader Key
+---
+
+The startup bootloader key feature reboots the keyboard into the bootloader if a specific key is held while powering on the keyboard. This is useful for keyboards which don't have a physical reset button or where the reset button is difficult to access.
+
+This is similar to the [`&bootloader` behavior](../behaviors/reset.md#bootloader-reset), except it doesn't take up a spot in your keymap since it overrides the behavior of the key just for a moment at startup. It also works on the peripheral side of a split keyboard even when it isn't connected to the central side.
+
+The startup bootloader key can be used as the primary way to enter the bootloader, or it can be a fallback in case you flash a broken keymap and can't access your `&bootloader` key.
+
+## Adding a Bootloader Key
+
+To enable this feature on a new board or shield, add a `zmk,startup-bootloader-key` node to your board's `.dts` file or shield's `.overlay` file and select which key will trigger it with the `key-position` property.
+
+You can also enable the feature for any keyboard by adding it to your `.keymap` file.
+
+```
+/ {
+    ...
+    bootloader_key: bootloader_key {
+        compatible = "zmk,startup-bootloader-key";
+        key-position = <0>;
+    };
+    ...
+};
+```
+
+:::info
+
+Key positions are numbered like the keys in your keymap, starting at 0. So, if the first key in your keymap is `Q`, this key is in position `0`. The next key (possibly `W`) will have position 1, etcetera.
```

Comment:
```suggestion
Key positions are numbered like the keys in your keymap, starting at 0. So, if the first key in your keymap is `Q`, this key is in position `0`. The next key (possibly `W`) will have position 1, etcetera. Note that the key index is affected by the chosen [matrix transform](../config/kscan#matrix-transform), so make sure to select an index that would be available in all of them if you define multiple matrix transforms.
```

Might be good to add a note like this one?

Reply:
This was a direct copy from the combos documentation. Do you think this should be added there too?

Reply:
That's a good point, the first part applies there as well.

For the second part, combos are different because they are defined by the user in the keymap, where the transform is known. Here this is directed towards users writing keyboard definitions, where they don't know what matrix transform will be chosen by the user.